docs/release/release-tasks.md

```diff
@@ -120,6 +120,7 @@ This comes down to changing occurrences of the old version to the new version, e
       Please note that both `InitWithKubernetesVersion` and `WorkloadKubernetesVersion` should be the highest mgmt cluster version supported by the respective Cluster API version.
 2. Update `create-local-repository.py` and `tools/internal/tilt-prepare/main.go`: `v1.4.99` => `v1.5.99`.
 3. Make sure all tests are green (also run `pull-cluster-api-e2e-full-main` and `pull-cluster-api-e2e-workload-upgrade-1-23-latest-main`).
+4. Remove unsupported release version of Cluster API. For example remove `v1.3` while working on `v1.6`.
```

Comment:
This is already covered under L~546. Similar to #9486 is the idea here to move responsibility of this from the CI lead to the release lead?

Reply:
This needs to be checked with @furkatgofurov7, I will wait for his response as he out for vacation.

Reply:
> This is already covered under L~546. Similar to #9486 is the idea here to move responsibility of this from the CI lead to the release lead?

@killianmuldoon this has to be under the RL responsibilities and part of the steps working on preparing the main branch work, as we need to remove [makefile target](https://github.com/kubernetes-sigs/cluster-api/pull/9097/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L520-L524) to generate e2e templates for unsupported release branch. Maybe we could clarify it a bit in the notes if that was not clear?

Reply:
I definitely think clarifying this is a good idea. This is the line in the CI tasks I was talking about:

> Remove tests for old release branches according to our policy documented in [Support and guarantees](https://github.com/chiukapoor/cluster-api/blob/issue-9485/CONTRIBUTING.md#support-and-guarantees) For example, let's assume we just created tests for v1.4, then we can now drop test coverage for the release-1.1 branch.

Maybe that should be clarified to only refer to the test-infra test config?

Reply:
>Maybe that should be clarified to only refer to the test-infra test config?

Agree, it has 2 parts:  changes needed in CAPI code base & test-infra repo, and above line refers to the latter. In that case, we could:

- leave the current notes where it is now but clarifying that the Makefile target for old branch has to be removed
- clarify the notes from CI tasks that it only refers to the test-infra repo changes

WDYT?

Reply:
Sounds good!

Reply:
@chiukapoor can you please update the PR accordingly?